### ISSUE TITLE
"Attachement" notifications look weird

### ISSUE THREAD

**Issue by afranke** [aperezdc/revolt#75]
When someone uploads a file to a room and you get a notification (direct conversation for instance), the text reads “Attachment”. It should rather be a proper sentence like “Rupert Monkey sent a file”.

**Comment by aperezdc** [comment]
Indeed, notifications for attachments certainly could be better. Somehow I had not noticed this (I have all but direct mention notifications disabled), so thanks @afranke for reporting it!
